core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -367,9 +374,26 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     }
   }
 
+  isImageRemoved(): boolean {
+    const tempDiv = this.renderer.createElement('div');
+    this.renderer.setProperty(tempDiv, 'innerHTML', this.editedContent.html);
+    return tempDiv.querySelector('oppia-noninteractive-image') === null;
+  }
+
+  get isUpdateDisabled(): boolean {
+    return this.startedEditing && this.isImageRemoved();
+  }
+
   updateSuggestion(): void {
     const updatedTranslation = this.editedContent.html;
     const suggestionId = this.activeSuggestion.suggestion_id;
+    if (this.initialHasImage && this.isImageRemoved()) {
```

Comment:
Here the condition is not considering the count of images in the translation suggestion. Imagine if there are 2 images in a suggestion and the reviewer removes one of them then this check can be bypassed. Consider matching the number of images.

Reply:
@masterboy376 I have added the frontend test cases. Thanks !